proxywasm/v1/grpc.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 func ProxyOpenGrpcStream(instance common.WasmInstance, grpcServiceData int32, grpcServiceSize int32,
-	serviceNameData int32, serviceNameSize int32, methodData int32, methodSize int32, returnCalloutID int32) int32 {
-
+	serviceNameData int32, serviceNameSize int32, methodData int32, methodSize int32, returnCalloutID int32,
+) int32 {
```

Comment:
`gofumpt -l -w .` is great at finding mild formatting issues https://github.com/mvdan/gofumpt